frontend/src/views/AdminAllGames.vue

```diff
@@ -67,22 +67,13 @@ export default {
       }
     ]
   }),
-  methods: {
-    async games() {
-      //this needs to be checked once backend is merged in
-      this.loading = true;
-      await axios.get('http://127.0.0.1:8080/game')
-        .then(response => {
-          this.users = response.data[0];
-          this.loading = false
-        }).catch(error => {
-          console.log(error.response.data);
-          this.dataLoading = false
-      });
-    },
+  asyncComputed: {
+    games: function () {
+      return axios.get("http://localhost:8080/game")
```

Comment:
axios is now a global property. I'll leave this for now, but you will be able to use axios by calling this.$axios() and you just need to uri.